src/rust/bitbox02-rust/src/hww/noise.rs

```diff
@@ -66,7 +66,7 @@ impl core::convert::From<()> for Error {
 /// Returns Err if anything goes wrong:
 /// - Invalid OP-code
 /// - Noise message in the wrong state (e.g. handshake before init, etc.).
-pub async fn process(usb_in: Vec<u8>, usb_out: &mut Vec<u8>) -> Result<(), Error> {
+pub(crate) async fn process(usb_in: Vec<u8>, usb_out: &mut Vec<u8>) -> Result<(), Error> {
```

Comment:
I don't think this is needed for `rust_noise_generate_static_private_key`. Why the change?

Reply:
`pub(crate)` means it's only public within the crate, so hww.rs can access it, but not the `-c` crate.

Reply:
oh, of course. because it's using `bitbox02_rust::hww::noise::BB02Random32`. good point.